<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSimStatus</class>
 <widget class="QWidget" name="MainSimStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>28</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="statusTip">
   <string>Main Sim Status</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label_sim_type">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(252, 252, 252)
</string>
     </property>
     <property name="text">
      <string> Main Sim </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_state">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_state_value">
     <property name="text">
      <string>&lt;none&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_time">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_time_value">
     <property name="text">
      <string>0000 00:00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_events">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Events:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_events_value">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_alert">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Alert:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_alert_indicator">
     <property name="text">
      <string>&lt;none&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
